ramin/js/es6/main.js

```diff
@@ -0,0 +1,68 @@
+// write an arrow function to add two numbers
+add = (num1,num2) => {
```

Comment:
To prevent the creation of a global variable, please define your variable with a keyword such as ‘var’, ‘let’, or ‘const’.

Reply:
thanks i fixed it